packages/flutter/test/painting/text_painter_test.dart

```diff
@@ -745,6 +745,13 @@ void main() {
 
     expect(lines.length, 4);
 
+    if (lines[1].hardBreak == true) {
```

Comment:
Can you leave a comment here with a link to the issue and a TODO to remove it?

Reply:
Already did! just forgot to push it.